sqlstore/src/main/resources/db/migration/V8810__Field_Units.sql

```diff
@@ -0,0 +1,10 @@
+ALTER TABLE `Library` ADD COLUMN `concentrationUnits` varchar(20) DEFAULT 'nM';
```

Comment:
These columns should be `NOT NULL`, I think. It won't be a problem since you're providing a default.